google/cloud/policytroubleshooter/iam/v3/.repo-metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "policytroubleshooter.googleapis.com",
+    "api_shortname": "policytroubleshooter",
+    "client_documentation": "https://cloud.google.com/cpp/docs/reference/policytroubleshooter/latest",
+    "distribution_name": "google-cloud-cpp",
+    "issue_tracker": "https://issuetracker.google.com/issues?q=componentid:690790%20status=open",
+    "language": "cpp",
+    "library_type": "GAPIC_AUTO",
+    "name_pretty": "Policy Troubleshooter API",
+    "product_documentation": "https://cloud.google.com/policy-intelligence/docs/troubleshoot-access",
```

Comment:
This link seems to indicate that the feature is "pre-GA" :roll_eyes: 

do we need to add an `experimental: true` field? or not generate it at all...

Reply:
It went GA this week. There may be a rollout delay on the documentation.